tests/basic/customer_issues_test.go

```diff
@@ -972,3 +976,174 @@ var _ = Describe("{ContainerCreateDeviceRemoval}", func() {
 		AfterEachTest(contexts)
 	})
 })
+
+// blockiSCSIInterfaceOnNode block and revert iscsi interface on Node
+func blockiSCSIInterfaceOnNode(n *node.Node, revertRules bool) error {
+
+	command := fmt.Sprintf("iptables -A INPUT -p tcp -s %v --dport 3260 -j DROP ", n.MgmtIp)
+	if revertRules {
+		command = fmt.Sprintf("iptables -D INPUT -p tcp -s %v --dport 3260 -j DROP", n.MgmtIp)
+	}
+	log.InfoD("Triggering command [%s] from Node [%v]", command, n.Name)
+	err := runCommand(command, *n)
+	if err != nil {
+		return err
+	}
+
+	command = fmt.Sprintf("iptables -A OUTPUT -p tcp -s %v --dport 3260 -j DROP ", n.MgmtIp)
+	if revertRules {
+		command = fmt.Sprintf("iptables -D OUTPUT -p tcp -s %v --dport 3260 -j DROP", n.MgmtIp)
+	}
+	log.InfoD("Triggering command [%s] from Node [%v]", command, n.Name)
+	err = runCommand(command, *n)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
```suggestion
// blockiSCSIInterfaceOnNode blocks or unblocks the iSCSI interface on a Node
func blockiSCSIInterfaceOnNode(n *node.Node, block bool) error {

	command := fmt.Sprintf("iptables -A INPUT -p tcp -s %v --dport 3260 -j DROP ", n.MgmtIp)
	if !block {
	command = fmt.Sprintf("iptables -D INPUT -p tcp -s %v --dport 3260 -j DROP", n.MgmtIp)
	}
	log.InfoD("Triggering command [%s] from Node [%v]", command, n.Name)
	err := runCommand(command, *n)
	if err != nil {
		return err
	}

	command = fmt.Sprintf("iptables -A OUTPUT -p tcp -s %v --dport 3260 -j DROP ", n.MgmtIp)
	if !block {
		command = fmt.Sprintf("iptables -D OUTPUT -p tcp -s %v --dport 3260 -j DROP", n.MgmtIp)
	}
	log.InfoD("Triggering command [%s] from Node [%v]", command, n.Name)
	err = runCommand(command, *n)
	if err != nil {
		return err
	}

	return nil
}
```

There is no change to the logic itself; however, you might consider renaming the argument for readability. This way, when we read


```go
err = blockiSCSIInterfaceOnNode(podNode, true)
```

it means that we are blocking the iSCSI interface on a node. Also,

```go
err = blockiSCSIInterfaceOnNode(podNode, false)
```

would then mean we are unblocking the iSCSI interface on a node. Currently, it is the other way around.